src/components/qa/subcomponents/AnswerListEntryItem.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {useState} from 'react';
+import {Card, CardMedia, Typography, Paper} from '@material-ui/core';
+
+
+
+export default function AnswerListEntryItem({ answer }) {
+  const [helpfullCount, setHelpfulCount] = useState(answer.helpfulness)
+  const [report, setReport] = useState(<u>Report</u>)
+  const toggleReport = () => {
+    setReport(<span style={{color: 'red'}}>Reported!</span>)
+  }
+  const [voteClicked, setVoteClicked] = useState(false)
+  const [voteCount, setVoteCount] = useState(answer.helpfulness)
+  const handleVoteClicked = () => {
+    if(voteClicked){
+      return <>Already Voted</>
+    } else {
+      setVoteCount(() => voteCount + 1)
+    setVoteClicked(true)
+    }
+    
```

Comment:
General bad styling (check linter).
Else is unneeded because of the return statement inside the `if` block.